runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/retries/policy/RetryPolicy.kt

```diff
@@ -6,13 +6,15 @@
 package aws.smithy.kotlin.runtime.retries.policy
 
 /**
- * A policy that evaluates a [Result] from a retry attempt and indicates the action a [RetryStrategy] should take next.
+ * A policy that evaluates a [Result] from a retry attempt and indicates the action a
+ * [aws.smithy.kotlin.runtime.retries.RetryStrategy] should take next.
  */
 public interface RetryPolicy<in R> {
     /**
      * Evaluate the given retry attempt.
      * @param result The [Result] of the retry attempt.
-     * @return A [RetryDirective] indicating what action the [RetryStrategy] should take next.
+     * @return A [RetryDirective] indicating what action the [aws.smithy.kotlin.runtime.retries.RetryStrategy] should
+     * take next.
```

Comment:
**Style**: Import `RetryStrategy` instead of fully qualifying it.

Reply:
`RetryStrategy` isn't used by the code anywhere though so fully qualifying it makes sense here (no unused imports)